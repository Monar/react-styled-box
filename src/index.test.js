import React from 'react';
import { Box } from './index';
import { shallow } from 'enzyme';

describe('Box', () => {
  it('default render', () => {
    const Component = shallow(<Box/>);
    expect(Component).toMatchSnapshot();
  });
});
