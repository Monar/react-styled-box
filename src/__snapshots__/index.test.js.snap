// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box default render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
  display="flex"
/>
`;
